packages/debugger-extension/src/index.ts

```diff
@@ -271,60 +275,148 @@ const sources: JupyterFrontEndPlugin<IDebugger.ISources> = {
+    const trackerMime = new WidgetTracker<Debugger.VariableRenderer>({
+      namespace: 'debugger/render-variable'
+    });
     const CommandIDs = Debugger.CommandIDs;
 
+    // Add commands
     commands.addCommand(CommandIDs.inspectVariable, {
       label: trans.__('Inspect Variable'),
       caption: trans.__('Inspect Variable'),
+      isEnabled: args =>
+        !!service.session?.isStarted &&
+        (args.variableReference ??
+          sidebar.variables.latestSelection?.variablesReference ??
+          0) > 0,
       execute: async args => {
-        const { variableReference } = args;
-        if (!variableReference || variableReference === 0) {
-          return;
+        let { variableReference, name } = args as {
+          variableReference?: number;
+          name?: string;
+        };
+
+        if (!variableReference) {
+          variableReference =
+            sidebar.variables.latestSelection?.variablesReference;
+        }
+        if (!name) {
+          name = sidebar.variables.latestSelection?.name;
         }
-        const variables = await service.inspectVariable(
-          variableReference as number
-        );
 
-        const title = args.title as string;
-        const id = `jp-debugger-variable-${title}`;
+        const id = `jp-debugger-variable-${name}`;
         if (
-          !variables ||
-          variables.length === 0 ||
+          !name ||
+          !variableReference ||
           tracker.find(widget => widget.id === id)
         ) {
           return;
         }
 
+        const variables = await service.inspectVariable(
+          variableReference as number
+        );
+        if (!variables || variables.length === 0) {
+          return;
+        }
+
         const model = service.model.variables;
         const widget = new MainAreaWidget<Debugger.VariablesGrid>({
           content: new Debugger.VariablesGrid({
             model,
             commands,
-            scopes: [{ name: title, variables }],
+            scopes: [{ name, variables }],
             themeManager
           })
         });
         widget.addClass('jp-DebuggerVariables');
         widget.id = id;
         widget.title.icon = Debugger.Icons.variableIcon;
-        widget.title.label = `${service.session?.connection?.name} - ${title}`;
+        widget.title.label = `${service.session?.connection?.name} - ${name}`;
         void tracker.add(widget);
-        model.changed.connect(() => widget.dispose());
+        const disposeWidget = () => {
+          widget.dispose();
+          model.changed.disconnect(disposeWidget);
+        };
+        model.changed.connect(disposeWidget);
+        shell.add(widget, 'main', {
+          mode: tracker.currentWidget ? 'split-right' : 'split-bottom',
+          activate: false
+        });
+      }
+    });
+
+    commands.addCommand(CommandIDs.renderMimeVariable, {
+      label: trans.__('Render Variable'),
+      caption: trans.__('Render Variable according to its mime type'),
```

Comment:
```suggestion
      caption: trans.__('Render variable according to its mime type'),
```

I would personally capitalise `MIME` but I see we already use it in lowercase elsewhere.